SmartDeviceLinkTests/DevAPISpecs/NSMutableDictionary+StoreSpec.m

```diff
@@ -0,0 +1,165 @@
+#import <Quick/Quick.h>
+#import <Nimble/Nimble.h>
+
+#import "NSMutableDictionary+Store.h"
+
+@interface TestObject : NSObject
+@property (nonatomic, strong, nullable) NSDictionary *json;
+- (instancetype)initWithDictionary:(NSDictionary *)json;
+@end
+
+@implementation TestObject
+- (instancetype)initWithDictionary:(NSDictionary *)json {
+    self = [super init];
+    if (self) {
+        self.json = json;
+    }
+    return self;
+}
+- (BOOL)isEqual:(id)object {
+    if (![object isKindOfClass:[TestObject class]]) {
+        return false;
+    }
+    TestObject *anotherObject = (TestObject *)object;
+    return [self.json isEqual:anotherObject.json];
+}
+@end
+
+QuickSpecBegin(NSMutableDictionary_StoreSpec)
+
+describe(@"A mutable dictionary with the storing category imported", ^{
+    __block NSMutableDictionary *testDictionary = nil;
+    __block NSString *testKey = @"Key";
+
+    beforeEach(^{
+        testDictionary = [[NSMutableDictionary alloc] init];
+    });
+
+    describe(@"adding a string", ^{
+        __block NSString *testObject = @"Object";
+
+        beforeEach(^{
+            [testDictionary sdl_setObject:testObject forName:testKey];
+        });
+
+        it(@"should return string", ^{
```

Comment:
@"should return string when called correctly"

Reply:
fixed in 6f291ca